dpi-client/src/main/java/no/difi/meldingsutveksling/dpi/client/DpiClientConfig.java

```diff
@@ -176,7 +178,7 @@ public UnpackJWT unpackJWT(BusinessCertificateValidator validator) {
 
     @Bean
     public BusinessCertificateValidator businessCertificateValidator() {
-        return BusinessCertificateValidator.of(properties.getDpi().getCertificate().getMode());
+        return BusinessCertificateValidator.of(Mode.valueOf(properties.getDpi().getCertificate().getMode()));
```

Comment:
Ser at BusinessCertificateValidator har en ".of(String path)" - hadde det vært en idé å benytte denne, og så droppe hele bruken av Mode? Da har man anledning til å overstyre valideringsoppskriften eksternt ved behov.

Reply:
Ja, vi kan det